code/game/objects/items/devices/compressionkit.dm

```diff
@@ -53,7 +53,7 @@
 		if(O.GetComponent(/datum/component/storage))
 			to_chat(user, "<span class='notice'>You can't make this item any smaller without compromising its storage functions!.</span>")
 			return
-		if(O.w_class == WEIGHT_CLASS_TINY)
+		if(O.w_class == WEIGHT_CLASS_TINY || O.w_class < initial(O.w_class))
```

Comment:
I believe this new condition should tell a different warning message since people from other codebases would think the same result, but getting this message would make them confused. (I mean, it shouldn't be merged with the same if condition.)

Reply:
Two unique error messages now in most recent commit